### PR TITLE
Dice: Fixes #3989 Set upper limit on max dice rolls and number of sides per dice

### LINKED ISSUE #3989: Dice: Set upper limit on max dice rolls and number of sides per dice
## Problem
When the Dice Goodie is given an extremely large number of sides for a dice roll, the DuckDuckGo UI breaks.

Eg. https://duckduckgo.com/?q=roll+6d12000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

## Solution

We should:
1. Limit the max number of simultaneous dice rolls to **10**
2. Limit the max number of sides for a given dice to **31**

## Get Started
- [ ] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---
https://duck.co/ia/view/dice

### PR DESCRIPTION
**Pull request by lernae**
## Description of new Instant Answer, or changes

This PR will set an upper limit on the max dice rolls to be 10 and max number of sides per side to be 31.

## Related Issues and Discussions

Fixes #3989

## People to notify

@mintsoft, @pjhampton, @moollaza 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/dice

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Dice.pm [hunk 1]
```diff
@@ -11,6 +11,9 @@ triggers start => "roll", "throw";
 zci answer_type => "dice_roll";
 zci is_cached => 0;
 
+use constant MAX_NUM_OF_DICES => 10;
```
Comment (mintsoft) [inline lib/DDG/Goodie/Dice.pm]: A bit of pedantry here, dice is already plural; die is the singular so this should be `MAX_NUM_OF_DICE`

**Comment by mintsoft** [comment]
The tests are totalled for a different reason, but they pass locally for me.

File: lib/DDG/Goodie/Dice.pm (hunk 1 above)
Comment (lernae) [inline lib/DDG/Goodie/Dice.pm]: @mintsoft good catch. :)  I'll correct it today.

**Comment by moollaza** [comment]
Deploying to Beta. Should be live in a few minutes for final test + merge 👍

**Comment by moollaza** [comment]
@lernae Thanks! This LGTM :+1:

**Pull request merged**